presto-main/src/test/java/com/facebook/presto/cost/TestOutputNodeStats.java

```diff
@@ -27,7 +27,6 @@
 
     @BeforeMethod
     public void setUp()
-            throws Exception
```

Comment:
Please add such commits using `--fixup=<commit_to_be_fixed>`
It spares me an effort of finding the commit I should squash it to.